<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="query_Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#query_Button{
color: rgb(255, 255, 255);
background-color: rgba(255, 11, 18, 220);
border:2px solid white;
border-radius:10px;
font: 13pt &quot;华文琥珀&quot;;
}
QPushButton#query_Button:disabled{
background-color: rgba(255, 98, 98, 150);
border:2px solid white;border-radius:10px;
}

QPushButton#query_Button:hover{ 
background-color: rgba(255, 11, 18, 140);
border:2px solid white;border-radius:10px;
}
 
QPushButton#query_Button:pressed{
background-color: rgba(255, 98, 98, 150);
border:2px solid white;border-radius:10px;
padding-left:6px;
padding-top:6px;
}
</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#clear_Button{
color: rgb(255, 255, 255);
background-color: rgba(255, 11, 18, 220);
border:2px solid white;
border-radius:10px;
font: 13pt &quot;华文琥珀&quot;;
}
 
QPushButton#clear_Button:hover{ 
background-color: rgba(255, 11, 18, 140);
border:2px solid white;border-radius:10px;
}
 
QPushButton#clear_Button:pressed{
background-color: rgba(255, 98, 98, 150);
border:2px solid white;border-radius:10px;
padding-left:6px;
padding-top:6px;
}
</string>
   </property>
   <property name="text">
    <string>清除屏幕</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#close_Button{
color: rgb(255, 255, 255);
background-color: rgba(255, 11, 18, 220);
border:2px solid white;
border-radius:10px;
font: 13pt &quot;华文琥珀&quot;;
}
 
QPushButton#close_Button:hover{ 
background-color: rgba(255, 11, 18, 140);
border:2px solid white;border-radius:10px;
}
 
QPushButton#close_Button:pressed{
background-color: rgba(255, 98, 98, 150);
border:2px solid white;border-radius:10px;
padding-left:6px;
padding-top:6px;
}
</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#pushButton
{
border-image: url(:/photo/p.png);
	
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#pushButton_2
{
    border-image: url(:/photo/baidu.png);
	
}

QPushButton#pushButton_2:pressed
{
    
	border-image: url(:/photo/baidu1.png);
}
QPushButton#pushButton_2:hover
{
    
	border-image: url(:/photo/baidu2.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输入的语言：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>451</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;翻译结果:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="text_send">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:11pt;&quot;&gt;请先选择你需要翻译的语言&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:11pt;&quot;&gt;本产品翻译结果由百度翻译提供&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:11pt;&quot;&gt;由于作者水平有限，有不足之处希望大家指出&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;by：安俊杰   &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;2019年10月1日 &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;请把你需要翻译的词组输入到下面框内:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="text">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 18pt &quot;新宋体&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="chinese_Button">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Agency FB&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>中文（Chinese）</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="english_Button">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Agency FB&quot;;

color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>英文（English）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="tcp_lineEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>网络连接成功</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
